<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>498</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="windowFilePath">
   <string>Настройки</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tabGeneral">
        <attribute name="title">
         <string>Общее</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabView">
        <attribute name="title">
         <string>Вид</string>
        </attribute>
        <widget class="QSplitter" name="splitter_2">
         <property name="geometry">
          <rect>
           <x>22</x>
           <y>22</y>
           <width>251</width>
           <height>117</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="lineWidth">
             <property name="text">
              <string>Толщина линии</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="handleSize">
             <property name="text">
              <string>Размер узла</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="numberSize">
             <property name="text">
              <string>Размер нумерации</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QSpinBox" name="spinLineWidth">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>50</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinHandleSize">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>50</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinNumberSize">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>50</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tabColor">
        <attribute name="title">
         <string>Цвет</string>
        </attribute>
        <widget class="QSplitter" name="splitter">
         <property name="geometry">
          <rect>
           <x>22</x>
           <y>11</y>
           <width>241</width>
           <height>204</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="handleColor">
             <property name="text">
              <string>Цвет узла</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="handleSelectColor">
             <property name="text">
              <string>Цвет выбранного узла</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="numberColor">
             <property name="text">
              <string>Цвет нумерации</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="backgroundColor">
             <property name="text">
              <string>Фон нумерации</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="rectColor">
             <property name="text">
              <string>Цвет прямоугольника</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="circleColor">
             <property name="text">
              <string>Цвет круга</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="polylineColor">
             <property name="text">
              <string>Цвет полилинии</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1px solid #777; 
background: #000000;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1px solid #777; 
background: #000000;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1px solid #777; 
background: #000000;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1px solid #777; 
background: #000000;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1px solid #777; 
background: #000000;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1px solid #777; 
background: #000000;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_7">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1px solid #777; 
background: #000000;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tabWorkdir">
        <attribute name="title">
         <string>Рабочая директория</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabPolyline">
        <attribute name="title">
         <string>Замыкание полилинии</string>
        </attribute>
        <widget class="QWidget" name="polylineLayoutHost">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>501</width>
           <height>138</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="doubleClickLMB">
            <property name="text">
             <string>По двойному клику ЛКМ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="LMBZeroPoint">
            <property name="text">
             <string>По одинарному клику ЛКМ на нулевой точке</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="holdingCtrl">
            <property name="text">
             <string>Поставить последнюю точку с зажатым Ctrl</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="keyC">
            <property name="text">
             <string>По клавише C</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
